Team9Project/src/com/indragie/cmput301as1/TagEditToClaimActivity.java

```diff
@@ -0,0 +1,99 @@
+/* 
+ * Copyright (C) 2015 Andrew Zhong
+ * 
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *  
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */	
+
+package com.indragie.cmput301as1;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.ActionMode;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+
+public class TagEditToClaimActivity extends TagAddToClaimActivity {
```

Comment:
It seems like there's quite a lot of duplicated code between this and `TagAddToClaimActivity`. Is there any way to reduce the duplication?